x-pack/plugins/alerting/common/routes/rule/apis/bulk_edit/schemas/v1.ts

```diff
@@ -38,7 +38,7 @@ const ruleSnoozeScheduleSchemaWithValidation = schema.object(
 );
 
 const ruleActionSchema = schema.object({
-  group: schema.string(),
+  group: schema.maybe(schema.string()),
```

Comment:
System actions do not have a group. Changing this to optional to allow system actions to be created without a group.